UnitySDK/Assets/ML-Agents/Scripts/InferenceBrain/BarracudaModelParamLoader.cs

```diff
@@ -241,7 +267,36 @@ static ModelActionType GetActionType(int isContinuousInt)
                     "You must set the Vector Observation Space Size to 0.");
             }
 
-            // TODO reenable checks there are enough Visual Observation Placeholder in the model.
+            // If there are not enough Visual Observation Input compared to what the
+            // sensors expect.
+            var visObsIndex = 0;
```

Comment:
There were removed in https://github.com/Unity-Technologies/ml-agents/pull/2731/files#diff-bbd30aae9fbe4211cf7e967224f2864dL242
Add them back here (and also check for not enough visual inputs, which was missing earlier).